src/test/java/teammates/storage/api/FeedbackResponseMonitorDbTest.java

```diff
@@ -0,0 +1,58 @@
+package teammates.storage.api;
+
+import java.util.List;
+
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.FeedbackResponseRecordAttributes;
+import teammates.common.exception.EntityAlreadyExistsException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.test.BaseComponentTestCase;
+
+/**
+ * SUT: {@link FeedbackResponseMonitorDb}.
+ */
+public class FeedbackResponseMonitorDbTest extends BaseComponentTestCase {
+    FeedbackResponseMonitorDb db = new FeedbackResponseMonitorDb();
+    long[] countTestData = new long[] { 105, 105, 105, 105, 105 };
+
+    private void populateTestData() throws EntityAlreadyExistsException, InvalidParametersException {
```

Comment:
Will there be a concern of non-deterministic timestamp resulting in inflexible testing framework?
Say in the future someone wants to write tests that test the timezone?

Reply:
Im getting the time from ```System.currentTimMillis()```, and generating timestamp from the pivot. The relationship between timestamp is relative, so timezone does not matter

Reply:
In this case, line 37 better use the exact same time. You should fix both to the same value.